src/plugins/lvm-dbus.c

```diff
@@ -905,6 +910,12 @@ static BDLVMLVdata* get_lv_data_from_props (GVariant *props, GError **error) {
         g_variant_unref (value);
     }
 
+    /* XXX: these values are currently not provided by the LVM DBus API */
```

Comment:
Aren't you getting the values for these from the LVM DBus API in `get_lv_data_from_props` changes from the other commit?